internal/cryptotest/aead.go

```diff
@@ -273,7 +273,7 @@ func TestAEAD(t *testing.T, mAEAD MakeAEAD) {
 					// Perturb the nonce and check for an error when Opening
 					alterNonce := make([]byte, aead.NonceSize())
 					copy(alterNonce, nonce)
-					alterNonce[len(alterNonce)-1] += 1
+					alterNonce[len(alterNonce)-1]++
```

Comment:
This file is a copy from upstream Go (see the README in this package). I would rather week it in sync.